ast/dml.go

```diff
@@ -775,7 +775,27 @@ func (n *OrderByClause) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
-// SelectStmt represents the select query node.
+type SelectStmtKind uint8
+
+const (
+	SelectStmtKindSelect SelectStmtKind = iota
+	SelectStmtKindTable
+	SelectStmtKindValues
+)
+
+func (s *SelectStmtKind) String() string {
+	switch *s {
+	case SelectStmtKindSelect:
+		return "SELECT "
+	case SelectStmtKindTable:
+		return "TABLE "
+	case SelectStmtKindValues:
+		return "VALUES "
```

Comment:
for consistency don't include the trailing space. (as a result add `ctx.WritePlain(" ")` after line 850.)

```suggestion
	case SelectStmtKindSelect:
		return "SELECT"
	case SelectStmtKindTable:
		return "TABLE"
	case SelectStmtKindValues:
		return "VALUES"
```

Reply:
fixed